exo-go/src/docker_helpers/index.go

```diff
@@ -17,9 +18,60 @@ func CatFileInDockerImage(c *client.Client, image, fileName string) ([]byte, err
 	return []byte(output), err
 }
 
+// ListRunningContainers passes a slice of the names of running containers
+// and error (if any) to the callback function
+func ListRunningContainers(c *client.Client, done func([]string, error)) {
```

Comment:
I had to use a callback here because this [step](https://github.com/Originate/exosphere/blob/cm-5-exo-run-cmd/exo-go/test_helpers/run_feature_context.go#L49) kept failing without the callback. I printed out the containers array, and saw the array growing as more tests with this step are run. This is very strange.